src/main/java/com/example/demo/ShortenUrlController.java

```diff
@@ -11,25 +13,27 @@
 public class ShortenUrlController {
```

Comment:
Let's have RestController annotation and general mapping here i.e. @RequestMapping("api/v1/url"). We should have version API for flexibility